spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/main/java/org/springframework/boot/configurationprocessor/TypeElementMembers.java

```diff
@@ -197,6 +197,10 @@ Map<String, List<ExecutableElement>> getPublicGetters() {
 		return Collections.unmodifiableMap(this.publicGetters);
 	}
 
+	Map<String, List<ExecutableElement>> getPublicSetters() {
+		return Collections.unmodifiableMap(this.publicSetters);
+	}
+
```

Comment:
copy of getPublicGetters, but here a generalization most likely is not worth it.